decidim-templates/config/locales/en.yml

```diff
@@ -57,6 +58,26 @@ en:
             title: New block user message template
           template_chooser:
             select_template: Select a template answer
+        proposal_answer_templates:
+          edit:
+            title: Edit proposal answer template
+          form:
+            hint1_html: "<strong>%{organization}</strong> will be replaced by the organization's name"
+            hint2_html: "<strong>%{name}</strong> will be replaced by the author's name"
+            hint3_html: "<strong>%{admin}</strong> will be replaced by the admin's name (the one answering the proposal)"
```

Comment:
(Not actionable, just a comment)

I really liked this string interpolation feature as we do in Newsletters. This is the typical thing that's good to have in the answers 👌🏽